kubetest/build.go

```diff
@@ -43,7 +43,7 @@ func (b *buildStrategy) Set(value string) error {
 		value = buildDefault
 	}
 	switch value {
-	case "bazel", "quick", "release":
+	case "bazel", "quick", "release", "all":
```

Comment:
maybe "nodocker" instead of "all"?

Reply:
SGTM

Reply:
or "host_go"?

Reply:
actually that sounds better. done.

Reply:
though I had to bikeshed that to `host-go` to match `quick-release` :-)